week2/The system of the student's score management. v1.0.cpp

```diff
@@ -0,0 +1,160 @@
+#include <stdio.h>
+
+void readscore(int n, long student_number[], int score[]);
+void print_total_and_average(int n, int score[]);

```

Comment:
I would suggest to declare parameters in primary types after those of composite types or array types. In this case,
```suggestion
void print_total_and_average(int score[], int n);
```